pkg/utils/avi_rest_utils.go

```diff
@@ -50,11 +54,30 @@ func SharedAVIClients() *AviRestClientPool {
 func NewAviRestClientPool(num uint32, api_ep string, username string,
 	password string) (*AviRestClientPool, error) {
 	var p AviRestClientPool
+	ctrlRootCa := os.Getenv("CTRL_ROOTCA")
```

Comment:
we will have to do a b64decode.

Reply:
I don't think so, Kubernetes does the de-coding and populates the environment variable accordingly. That is how the password/username is also getting handled.

Reply:
Ok.